code/modules/library/space_catalog.dm

```diff
@@ -39,21 +39,22 @@ GLOBAL_LIST_EMPTY(space_catalog_buffer)
 			if(!P2.station) // Ignore station-less planets
 				continue
 			// Add the station to the buffer
-			dat += "<h3>[P2.name] - [S2.alignment] ([round(dist,0.1)]ly away)</h3><br>"
+			dat += "<h3>[S2.alignment]-aligned station [P2.station.module.name] at [P2.name]([round(dist,0.1)]ly away)</h3><br>"
```

Comment:
Also check_tick this.

If you can move it to initialize but I'm not sure if books will handle that right so test it.